com.github.thetumultuousunicornofdarkness.cpu-x.yaml

```diff
@@ -0,0 +1,67 @@
+app-id: com.github.thetumultuousunicornofdarkness.cpu-x
+runtime: org.gnome.Platform
+runtime-version: "44"
```

Comment:
45 is the current version.

Reply:
Done in 28981c5ab35813a06496c145619e7447eeea1aef.